Jint/Native/Function/ClassDefinition.cs

```diff
@@ -359,6 +364,10 @@ public ClassStaticBlockFunction(StaticBlock staticBlock) : base(Nodes.StaticBloc
         {
             var methodDef = method.DefineMethod(obj);
             methodDef.Closure.SetFunctionName(methodDef.Key);
+            // TODO currently in the SourceText retrieved from Esprima, the method name is incorrect, so for now, use a string replacement.
```

Comment:
Wouldn't it make more sense to fix the wrong name on Esprima side?

Reply:
Because I don't know how to fix it in Esprima (

Reply:
There are a lot of example test cases. You would just need to check the produced AST and that the identifier is correct. Minimal test case helps.

Reply:
I tried it, and after debugging for a while, I gave up. Maybe you can take a look? I'll try it again later.

Reply:
I generally have a lot more free time for OSS if someone takes care of my child and cleans the house.